formats/policy/frontend/schema.json

```diff
@@ -99,6 +99,9 @@
                     }
                   }
                 }
+              },
+              "result_metadata_name_key": {
```

Comment:
I'm happy enough for the changes to go through in one PR. That's the way I've always done it.


> On 27 Apr 2015, at 10:21, David Singleton <notifications@github.com> wrote:
> 
> In formats/policy/frontend/schema.json:
> 
> > @@ -99,6 +99,9 @@
> >                      }
> >                    }
> >                  }
> > +              },
> > +              "result_metadata_name_key": {
> Yep, i'm putting that PR together now.
> 
> I'd assumed the process was get the schema agreed first - then other PRs (though i cheated with the FF one, as it's got more than just a schema update.
> 
> —
> Reply to this email directly or view it on GitHub.
>